example_configs/weblogic.yml

```diff
@@ -0,0 +1,60 @@
+---
+jmxUrl: service:jmx:iiop:///jndi/iiop://localhost:7001/weblogic.management.mbeanservers.runtime
+username: weblogic
+password: XXXXXXX
+lowercaseOutputName: true
+lowercaseOutputLabelNames: true
+whitelistObjectNames:
+  - "com.bea:Name=*,Type=ServerRuntime"
+  - "com.bea:ServerRuntime=*,Type=ApplicationRuntime,*"
+  - "com.bea:ServerRuntime=*,Type=JDBCDataSourceRuntime,*"
+  - "com.bea:ServerRuntime=*,Type=JMSDestinationRuntime,*"
+  - "com.bea:ServerRuntime=*,Type=SAFRemoteEndpointRuntime,*"
+  - "com.bea:ServerRuntime=*,Type=ThreadPoolRuntime,*"
+
+rules:
+  - pattern: "^com.bea<ServerRuntime=.+, Name=(.+), Type=JDBCDataSourceRuntime><>(.+): (Running|Suspended|Shutdown|Overloaded|Unknown)$"
+    attrNameSnakeCase: true
+    name: weblogic_datasource_$2
+    value: 1
+    labels:
+      name: $1
+      state: $3
+
+  - pattern: "^com.bea<ServerRuntime=.+, Name=(.+), Type=JDBCDataSourceRuntime><>(.+):"
```

Comment:
Can you trim the `Count` off the end of these, where present? That should only be on Summary/Histograms, not Gauges.